### PR TITLE
ODK2 - App Designer Prereq Changes

### PR DESCRIPTION
**Pull request by smcooper1920**
#### What is included in this PR? 
Changes to App Designer Prerequisites to enhance windows directions

### CODE REVIEW

**Comment by yanokwa** [comment]
Thanks for sending this in @smcooper1920!

File: odk2-src/app-designer-prereqs.rst [hunk 1]
```diff
@@ -135,7 +144,7 @@ To install the Android SDK:
   5. Wait for the install of the SDK Tools to complete. Windows will need to manually run the :file:`.exe` file previously downloaded to start installation.
   6. Run the SDK Manager
 
-    - On Windows, it is available in the :guilabel:`Start Menu` under Android SDK Tools
+    - On Windows, it is available in the :guilabel:`Start Menu` under Android SDK Tools (If you have an error where the packages fail to install then you should run Android SDK as an Administator).
```
Comment (jbeorse) [inline odk2-src/app-designer-prereqs.rst]: Administrator is spelled incorrectly (which is why the build is failing).

File: odk2-src/app-designer-prereqs.rst [hunk 2]
```diff
@@ -116,6 +121,10 @@ If the above command is unsuccessful, some machines may need to append :command:
 
 Should display the installed version of :program:`grunt`. For example the version might be ``grunt-cli v1.2.0``
 
+.. warning::
+
+  If :program:`grunt` is not found, then you need to add it to the *PATH* variable of your system.
```
Comment (jbeorse) [inline odk2-src/app-designer-prereqs.rst]: I would modify this slightly to say "then you may need to add it". I agree that this is a likely problem, but I wouldn't want to declare that this is the definite solution.

File: odk2-src/app-designer-prereqs.rst (hunk 1 above)
Comment (jbeorse) [reply odk2-src/app-designer-prereqs.rst]: I think this should be in its own `warning` box.

File: odk2-src/app-designer-prereqs.rst [hunk 3]
```diff
@@ -90,6 +90,11 @@ After installing NodeJS, open a :program:`terminal` (which you can do by clickin
 
   outputs a message telling you permission is denied, then you will have to change the ownership of the :file:`/usr/local/` and :file:`/usr/local/bin/` directories. On Mac, follow the `instructions to take ownership <http://osxdaily.com/2013/04/23/change-file-ownership-mac-os-x/>`_ of these directories, or to at least give yourself read permission. On other Unix systems, use the :program:`chown` command or the user-interface appropriate to your distribution to do so.
 
+
+.. warning::
+
+  Add the location of the :program:`npm` folder to the *PATH* variable of your system if subsequent calls to access grunt in the next step fail. Ex. :file:`C:\\Users\\[username]\\AppData\\Roaming\\npm`
```
Comment (jbeorse) [inline odk2-src/app-designer-prereqs.rst]: I think this should be clearer that its a Windows tip (separate from the Mac/Unix tips above). I think you should create a new section "For Windows" that includes this tip.

**Comment by jbeorse** [comment]
There hasn't been any movement on this for a week. I'm happy to push the changes to resolve these issues if that's ok with you @smcooper1920

**Comment by yanokwa** [comment]
@jbeorse I think you can go ahead and branch off this and send in a PR!

**Comment by jbeorse** [comment]
I have submitted a new PR with the changes listed in https://github.com/opendatakit/docs/pull/791. This PR can be closed.

**Pull request closed without merging**